org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -10,10 +10,20 @@
     <method name="turnOff"/>
     
     <!-- Get the current ambient temperature -->
-    <method name="getCurrentTemperature">
+    <method name="getAmbientTemperature">
+      <!-- Temperature scale:  'f' for Fahrenheit. 'c' for celsius -->
+      <arg name="scale" type="s" direction="in" />
       <arg name="temp" type="d" direction="out" />
     </method>
 
+    <!-- Set the current ambient temperature -->
+    <method name="setTargetTemperature">
```

Comment:
My understanding was, they are min/max values for heating/cooling.. 

Tariq, can you pls clarify?

The way I mapped..

set/getHeatingSetpoint
    "target_temperature_high_c": 26,
    "target_temperature_high_f": 79,

set/getCoolingSetpoint
    "target_temperature_low_c": 19,
    "target_temperature_low_f": 66,

get/setTargetTemperature (new method that is added). I need to add 'get' as well.
   "target_temperature_c": 24,
    "target_temperature_f": 75,

getAmbientTemperature (formerly getCurrentTemperature)
    "ambient_temperature_c": 11,
    "ambient_temperature_f": 52,




---
In reply to: [68280848](https://github.com/openT2T/translators/pull/22#discussion_r68280848) [](ancestors = 68280848)